src/ontology/config/remove-annotations-before-release.txt

```diff
@@ -0,0 +1,5 @@
+http://purl.obolibrary.org/obo/mondo#CLINGEN_PREFERRED
+http://purl.obolibrary.org/obo/mondo#clingen_preferred
+http://purl.obolibrary.org/obo/mondo#excluded_from_qc_check
+http://purl.obolibrary.org/obo/mondo#confidence
+MONDO:preferredExternal
```

Comment:
The last step will never work (removing "preferredExternal"

Reply:
roger that